### ISSUE TITLE
Chrome version 74

### ISSUE THREAD

**Issue by Glecun** [searls/jasmine-maven-plugin#336]
Hi,

With the latest Chrome version I've got this error when running the tests:
```
[ERROR] Failed to execute goal com.github.searls:jasmine-maven-plugin:3.0-alpha-01:test (default) on project vulcain-web-flow: The jasmine-maven-plugin encountered an exception: session not created: This version of ChromeDriver only supports Chrome version 74
[ERROR] (Driver info: chromedriver=74.0.3729.6 (255758eccf3d244491b8a1317aa76e1ce10d57e9-refs/branch-heads/3729@{#29}),platform=Linux 4.15.0-45-generic x86_64) (WARNING: The server did not provide any stacktrace information)
[ERROR] Command duration or timeout: 195 milliseconds
[ERROR] Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'
[ERROR] System info: host: 'VID02324', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.15.0-45-generic', java.version: '1.8.0_144'
[ERROR] Driver info: driver.version: ChromeDriver
```

Is it possible to **set a specific version of Chrome** instead of using the latest version ? (And I do not want to use a binary)

Thanks in advance

**Comment by Glecun** [comment]
It seems to be linked to this recent commit in webdrivermanager that add the lastest chromedriver version (74) : https://github.com/bonigarcia/webdrivermanager/commit/6ddcaa6edf6697edfa02584126aba33c7fcdabe8

I see that webdrivermanager has a "version" method which specifies a concrete version.
https://github.com/bonigarcia/webdrivermanager#webdrivermanager-api
Can I use this method with jasmine-maven-plugin ?

**Comment by jasonmclaren** [comment]
I get this error too. Note that version 74 is currently in the canary/dev Chrome channels; beta/stable are still on version 73. It would be nice to be able to track stable rather than dev.

**Comment by georges-yam** [comment]
As a workaround, it should be possible to force a specific version of Driver using a system property from webdrivermanager called `wdm.chromeDriverVersion`.

For instance by adding `-Dwdm.chromeDriverVersion=73.0.3683.68` in the command line arguments. I can confirm it works on my end.

The relevant classes in webdrivermanager are `io.github.bonigarcia.wdm.WdmConfig` and `io.github.bonigarcia.wdm.ChromeDriverManager`.

**Comment by klieber** [comment]
This should be fixed in [3.0-beta-01](https://github.com/searls/jasmine-maven-plugin/releases/tag/jasmine-maven-plugin-3.0-beta-01).
